### DISCUSSION TITLE
Setting default `lang` in cb.code component as `ftd`

### DISCUSSION THREAD

**Discussion by gargajit** [fastn-stack/fastn#951] [Ideas & RFCs]
@Heulitig#6500 and I propose that we can give the default value as `ftd` to the `lang` property of the code component.

**Possible problems:**
- If people use any other code, they have to mention it, and out of habit, people can forget to add the property, eg: `lang: java` etc.

- The first-time users might not understand why their code is not highlighted based on their language because they won't get an error since by default lang is `ftd`.

**Solution:**
We can mention the language at the bottom right when the code block is rendered, just like we show the file name in the code and rendered components when the download property is used.

**Comment by amitu** [comment]
I generally like this. Let's do this.

**Comment by gsalunke** [comment]
@gargajit the changes suggested are implemented in https://github.com/fastn-community/code-block
via commit:
https://github.com/fastn-community/code-block/commit/efad8ae74e8ab2c281e8950e3774c4e148076841

Please, review: https://fastn-community.github.io/code-block/docs/code/
